HotelBooking/HotelBooking.DataAccessLayer/Entities/Guest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+public class Guest
+{
+    public string Firstname { get; set; }
+    public string Lastname { get; set; }
+    public string Email { get; set; }
+    public long PhoneNumber { get; set; }
+    private int age; //data urodzenia ewentualnie
```

Comment:
Zastanawiam się czy potrzebny nam wiek, czy może bardziej data urodzenia. Jak kojarzę jak się rezerwuje jakieś pokoje w hotelu to raczej nie wpisuje się wieku tak w liczbach tylko jednak datę urodzenia
Dałbym datę urodzenia o typie DateTime